README.md

```diff
@@ -32,73 +32,131 @@ live preview will be added to each PR.
+  See:[script/sync_repo.clj](https://github.com/metabase/docs.metabase.github.io/blob/documenting-manual-workflow-runs/script/sync_repo.clj#L16-L31).
+  These are needed to avoid conflicts with the marketing site.
 
-### [Process Docs Changes](.github/workflows/process_docs_changes.yml)
+# Process Docs Changes Workflow
 
-Notable steps:
+This workflow handles all documentation updates triggered by changes in the main
+`metabase/metabase` repository, including SDK Docs generation.
 
-- `Update docs for branchname`:
-  - Downloads docs from metabase/metabase, adds frontmatter, and builds SDK docs
-    using cljs compiler.
+## Running Manually
 
-- Builds Jekyll site
+It can be nice to run these by-hand, here's where you can trigger it manually:
+[Process Docs
+Changes](https://github.com/metabase/docs.metabase.github.io/actions/workflows/process_docs_changes.yml).
 
-- Lints markdown, styles, scripts, and links.
 
-- Opens PR with changes to `master` for the `_site` (html/js/css), and `_docs`
-  (markdown) directories.
+### Process Docs Changes Parameters
 
-#### Steps required for a faithful build
+| Parameter | Required | Type | Default | Description |
+|-----------------|----------|--------|-----------------|-------------------------------------------------------------------------------------|
+| `source_branch` | ✅ | string | - | The feature/source branch from metabase/metabase that triggered this workflow |
+| `target_branch` | ✅ | string | - | The target branch in metabase/metabase (`master` or `release-x.MM.x` for backports) |
+| `annotation` | ❌ | string | `"auto-build"` | comment displayed in the build queue and PR title for data lineage |
+| `dispatch_type` | ❌ | choice | `"docs_update"` | Action type: `docs_update`, `docs_merge`, or `docs_close` |
+| `pr_number` | ❌ | string | `-` | adds a link to the docs PR to the original PR in metabase/metabase |
+| `update_dirs` | ❌ | string | `""` | comma delimited string of extra dirs to git add during `docs_update` |
 
-- Since we've split up the site into 2 jekyll instances, we cannot rely on
-  htmlproofer to check links from the docs to the marketing site. So
-  `analyze_links.clj` checks any missing links against `metabase.com`.
-  
-- Copies over control directories from marketing repo: `_data`, `_includes`,
-  `_layouts`, `_plugins`, and `_sass`. These are needed so we get a faithful
-  docs build.
+#### Tips and Best Practices
+
+##### Annotations
+- Use descriptive annotations to identify workflows in the build queue and PR
+  titles
+- Keep them concise since they appear in PR titles
+
+##### Branch Naming
+- Source branch must match exactly what's in metabase/metabase
+- Target branch is typically `master`
+- Use `release-x.MM.x` format for backports (e.g., `release-0.50.x`)
+- The source branch must exist in metabase/metabase when running this workflow.
+  If the source branch has been deleted (e.g., after merging the original PR),
+  the workflow will fail.
```

Comment:
>  If the source branch has been deleted (e.g., after merging the original PR),
  the workflow will fail.

What do we do in this scenario?

Reply:
```suggestion
  If the source branch has been deleted (e.g., after merging the original PR),
  the workflow will fail. So follow the link to the main repo PR, and restore
  the branch there.
```